lua_cjson.c

```diff
@@ -75,6 +75,8 @@
 #define DEFAULT_DECODE_INVALID_NUMBERS 0
 #endif
 
+static const char *json_empty_array = "json_empty_array";
```

Comment:
We can avoid the array value `"json_empty_array"` here to save some memory. All we need now is just an address.